classes/02class/exercises/c02-network01/mendoncaj/README.md

```diff
@@ -0,0 +1,47 @@
+# c02-network01
+
+## Commands Execution Output
+
+- Answers for `10.0.0.1/24`:
+
+  - IP in binary: `00001010.00000000.00000000.00000001`
+  - subnet mask in binary: `11111111.11111111.11111111.00000000` 
+  - subnet mask in decimals: `255.255.255.0`
+  - private/public: `private` ( Class A )
```

Comment:
Might be a bit of technical annoyance so don't worry too much.

`10.0.0.0/8` is a Class A network. Once you break it down into subnets you cannot call them "Class A subnets" anymore.

Same idea for the classes on the IPs below